app/ui/components/accounts-list/accounts-list.jsx

```diff
@@ -6,15 +6,28 @@ import { removeAccount } from '../../store/actions';
 import { floorLocalFloatAmount, floorLocalAmountToK } from '../../../helpers/balance';
 import './accounts-list.scss';
 
+import { getAccounts } from '../../services/scorum';
+
 class AccountsListUI extends Component {
   constructor() {
     super();
 
     this.removeAccount = this.removeAccount.bind(this);
   }
 
-  componentDidMount() {
-    // ...
+  async componentDidMount() {
+    const { accounts } = this.props;
+    const keys = Object.keys(accounts);
+
+    if(!keys)
+      return;
+
+    const fetchedAccounts = await getAccounts(keys);
+    keys.forEach(k=>{
+      accounts[k].vp = fetchedAccounts.find(x=>x.name == k).voting_power;
```

Comment:
Use clear variables: "k" => "key", 
NOT: fetchedAccounts.find(x=>x.name == k).voting_power
=> fetchedAccounts.find(account=>account.name == key).voting_power

etc..

Reply:
And always use Strict equality "==="